pkg/asset/installconfig/platformpermscheck.go

```diff
@@ -66,6 +70,22 @@ func (a *PlatformPermsCheck) Generate(dependencies asset.Parents) error {
 			permissionGroups = append(permissionGroups, awsconfig.PermissionCreateHostedZone)
 		}
 
+		var ec2RootVolume = aws.EC2RootVolume{}
+		var awsMachinePoolUsingKMS, masterMachinePoolUsingKMS bool
+		if ic.Config.AWS.DefaultMachinePlatform != nil && ic.Config.AWS.DefaultMachinePlatform.EC2RootVolume != ec2RootVolume {
+			logrus.Debug("Using KMS key from default machine pool")
```

Comment:
if len(ic.Config.AWS.DefaultMachinePlatform.EC2RootVolume.KMSKeyARN) == 0, we would not be using KMS key from this machine pool. So, I think it is OK to remove above debug msg.